components/properties/src/lib.rs

```diff
@@ -23,19 +23,12 @@
 //!
 //! // A binary property as a `CodePointSetData`
 //!
-//! let data = sets::load_emoji(&icu_testdata::unstable())
-//!     .expect("The data should be valid");
-//! let emoji = data.as_borrowed();
-//!
-//! assert!(emoji.contains('🎃')); // U+1F383 JACK-O-LANTERN
-//! assert!(!emoji.contains('木')); // U+6728
+//! assert!(sets::emoji().contains('🎃')); // U+1F383 JACK-O-LANTERN
+//! assert!(!sets::emoji().contains('木')); // U+6728
```

Comment:
Nit, here and elsewhere (optional): I have a slight preference for keeping the intermediate object stored in its own variable since it is closer to what the code looks like if you use the explicit data provider version of it